.github/workflows/moodle-plugin-ci.yml

```diff
@@ -1,6 +1,6 @@
 name: Moodle Plugin CI
 
-on: [push, pull_request]
+on: [push] # [push, pull_request] # TODO: REENABLE BEFORE PR
```

Comment:
I think we can leave PR disabled. It just makes things slower with no real upside